integrations/zendesk/src/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { IntegrationContext, IntegrationDefinition } from '@botpress/sdk'
+import type * as botpress from '.botpress'
+
+type IntegrationDef = ConstructorParameters<typeof IntegrationDefinition>[0]
+export type EventDefinition = Extract<IntegrationDef['events'], {}>['string']
+export type ActionDefinition = Extract<IntegrationDef['actions'], {}>['string']
+export type ChannelDefinition = Extract<IntegrationDef['channels'], {}>['string']
+export type IntegrationCtx = IntegrationContext<botpress.configuration.Configuration>
+
+export type Implementation = ConstructorParameters<typeof botpress.Integration>[0]
```

Comment:
**nitpick:** there is already a type generated for that: `botpress.IntegrationProps`